commit b480e3db3a01b75376fa6b83e5b01efe104ccaec
Author: Darrell Anderson <humanreadable@yahoo.com>
Date:   1331957353 -0500

    Change location where Kaffeine stores temporary pipe files from $HOME to
    the more appropriate $TDEHOME/tmp-$HOSTNAME.

diff --git a/kaffeine/src/input/dvb/dvbpanel.cpp b/kaffeine/src/input/dvb/dvbpanel.cpp
index 365b7ac..969b62b 100644
--- a/kaffeine/src/input/dvb/dvbpanel.cpp
+++ b/kaffeine/src/input/dvb/dvbpanel.cpp
@@ -1065,7 +1065,8 @@ void DvbPanel::setConfig()
 		connect( d, TQT_SIGNAL(isRecording(bool)), this, TQT_SLOT(setRecordLed(bool)) );
 		connect( d, TQT_SIGNAL(playDvb()), this, TQT_SLOT(pipeOpened()) );
 	}
-	fifoName = TQDir::homeDirPath()+"/.kaxtv.ts";
+// 	fifoName = TQDir::homeDirPath()+"/.kaxtv.ts";
+	fifoName = KGlobal::dirs()->saveLocation("tmp")+"/.kaxtv.ts";
 	TQFile f( fifoName );
 	if ( f.exists() )
 		f.remove();
@@ -1073,7 +1074,8 @@ void DvbPanel::setConfig()
 		perror( fifoName.latin1() );
 		fifoName = "";
 	}
-	fifoName1 = TQDir::homeDirPath()+"/.kaxtv1.ts";
+// 	fifoName1 = TQDir::homeDirPath()+"/.kaxtv1.ts";
+	fifoName1 = KGlobal::dirs()->saveLocation("tmp")+"/.kaxtv1.ts";
 	TQFile f1( fifoName1 );
 	if ( f1.exists() )
 		f1.remove();
